signing_service/signing_service/utils.py

```diff
@@ -110,7 +113,43 @@ def __call__(  # type: ignore
         setattr(namespace, self.dest, self.const)
 
 
-class Notifier(object):
+class Notifier():
```

Comment:
As stated above - we need two `Notifier` classes

Reply:
Done